screens/Profile/profile/ProfileScreen.jsx

```diff
@@ -40,9 +41,15 @@ const Profile = ({navigation}) => {
 
   const accessCamera = async () => await requestCameraPermission(selectFile);
 
+  const renderSwitchValue = (item) => {
+    if(showHide)  loading ?  <SkeletonPhotos /> : <Photos item={item} />
+    else loading ? <SkeletonPosts /> : <Posts item={item}/>
```

Comment:
not readable code, please use if/else with { }

Reply:
Updated.